### PR TITLE
Exercise 3

### PR DESCRIPTION
**Pull request by LeoCie**

### CODE REVIEW

File: todo_app/data/trello_items.py [hunk 1]
```diff
@@ -3,27 +3,27 @@
...
 
 def delete_item(cardId):
+    board, authParams = getBoardAndAuthParams()
     return requests.delete(trello_base_url + f'/cards/{cardId}{authParams}')
 
 def create_item(title):
+    board, authParams = getBoardAndAuthParams()
     lists = makeGetRequest(trello_base_url + f'/boards/{board}/lists{authParams}')
     todoListId = [list for list in lists if list['name'] == 'To Do'][0]['id']
     requests.post(trello_base_url + f'/cards{authParams}&name={title}&idList={todoListId}')
```
Comment (BenRamchandani) [inline todo_app/data/trello_items.py:29]: What if I want to call my to do item `Some Title&desc=Injected Description`?

File: todo_app_tests/integrationtests/test_app.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    # Create the new app.
+    test_app = app.create_app()
+    # Use the app to create a test_client that can be used in our tests.
+    with test_app.test_client() as client:
+        yield client
+
+def test_index_page(monkeypatch, client):
+    monkeypatch.setattr(requests, 'get', get_lists_stub)
+    response = client.get('/')
+    assert response.status_code == 200
```
Comment (BenRamchandani) [inline todo_app_tests/integrationtests/test_app.py:21]: Would be nice to test at least that the test data has come through, e.g.
```python
assert 'Test card' in response.data.decode()
```

**Review by BenRamchandani — COMMENTED** [review]
The tests looks good, only thing is you haven't installed pytest with poetry, so it relies on pytest being installed globally. Run `poetry add --dev pytest` and commit the changes before you merge.

I've left a couple of other optional comments, and if you have time I encourage you to have a go at the Selenium stretch goal as it makes some of the later exercises more involved. If not that's fine, go ahead and merge once you've got pytest tracked by poetry.

File: todo_app/data/trello_items.py (hunk 1 above)
Comment (LeoCie) [reply todo_app/data/trello_items.py:31]: I have now sanitized the input

File: todo_app_tests/integrationtests/test_app.py (hunk 2 above)
Comment (LeoCie) [reply todo_app_tests/integrationtests/test_app.py:21]: Added this line

**Pull request merged**